docs/rules/prefer-tacit.md

```diff
@@ -82,3 +88,7 @@ Because when assuming types, false positives may be found, it's recommended to s
 ### `ignorePattern`
 
 See the [ignorePattern](./options/ignore-pattern.md) docs.
+
+### `ignoreImmediateMutation`
```

Comment:
This option was not mentioned. This is part of the [`--rule-doc-section-options`](https://github.com/bmish/eslint-doc-generator#configuration-options) option.